docs/OTAnalytics/otconfig.md

```diff
@@ -0,0 +1,34 @@
+# OTConfig
```

Comment:
Two questions regarding this file: 

(1) Is this page intentonally not included in the navigation?

(2) Is OTConfig publicly available (meaning: Does it make sense to publish this part of the documentation on Github?)

Reply:
FYI ... I didn´t review this page until now - should I?

Reply:
> Two questions regarding this file:
> 
> (1) Is this page intentonally not included in the navigation?
> 
> (2) Is OTConfig publicly available (meaning: Does it make sense to publish this part of the documentation on Github?)

(1) This is intentionally not included in the navigation. The otanalytics documentation is based on the internal documentation of OTConfig. Therefore, the remaining parts are left here. We can integrate this into the documentation at a later point.

(2) No, it is an internal documentation provided to clients of platomo. It describes the interaction with the platomo team.

Reply:
> FYI ... I didn´t review this page until now - should I?

It is not necessary to review it at the moment. We do not publish it.

Reply:
Hm. Having the Markdown souce code in this repository is also kind of public :) I don´t fully understand, why we have it here. But that´s ok to me, I just wanted to mention it to you.